### PR TITLE
Create nexusmods.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#11082

### CODE REVIEW

File: src/chrome/content/rules/nexusmods.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="nexusmods.com">
+	<target host="nexusmods.com" />
+	<target host="www.nexusmods.com" />
+	<target host="filedelivery.nexusmods.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/nexusmods.com.xml]: forums.nexusmods.com
wiki.nexusmods.com

Comment (koops76) [inline src/chrome/content/rules/nexusmods.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Can you merge this with `Forums.nexusmods.com.xml` please?

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**